src/amp/server/render.tsx

```diff
@@ -16,8 +16,8 @@ export const render = (
     try {
         // TODO remove when migrated to v2
         const CAPI = validateV2(body);
-        const linkedData = CAPI.linkedData;
-        const config = CAPI.config;
+        const {linkedData} = CAPI;
+        const {config} = CAPI;
```

Comment:
I much prefer destructuring so this rule is great but this pattern works much better when grouped into one statement. Maybe something we can boyscout as we go along?